src/prototype_creep_resources.js

```diff
@@ -280,6 +280,26 @@ Creep.prototype.pickupEnergy = function() {
     }
   }
 
+  // FIND_TOMBSTONES and get them empty first
+  const tombstones = this.pos.findInRange(FIND_TOMBSTONES, 1, {
+    filter: (t) => {
+      return t.creep.my;
+    },
+  });
+  if (tombstones.length > 0) {
+    const creep = this;
+    _.map(tombstones, (t) => {
+      if (t.store[RESOURCE_ENERGY] > 0) {
```

Comment:
Why only energy?

Reply:
it was in the getEnergy function... but a onlyEnergy flag i would like, or why would a sourcer on his way pick up resources?

Reply:
Yeah, good point